src/python/tests/core/platforms/android/device_test.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for device functions."""
+
+import unittest
+
+from platforms.android import adb
+from platforms.android import device
+from system import environment
+from tests.test_libs import helpers as test_helpers
+from tests.test_libs import test_utils
+
+
+@test_utils.android_device_required
+class BaseDeviceTest(unittest.TestCase):
```

Comment:
Could we move this class to some place in test_libs?

Reply:
created android_helpers.py similar to untrusted_runner_helpers.py in test_libs.